pkg/webhook/resources/virtualmachine/mutator.go

```diff
@@ -3,31 +3,42 @@ package virtualmachine
 import (
 	"encoding/json"
 	"fmt"
+	"strings"
 
 	admissionregv1 "k8s.io/api/admissionregistration/v1"
 	v1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog/v2"
```

Comment:
prefer to use logrus like other logs in the webhook server
use `logrus.Debugf` instead of `klog.Infof`